cni/api/api.go

```diff
@@ -6,9 +6,15 @@ import (
 	"os"
 
 	"github.com/Azure/azure-container-networking/cni/log"
+	"github.com/Azure/azure-container-networking/zaplog"
```

Comment:
We are importing two log module for the same file ?
Can we try to do something like `zaplog` as the global one. And then the for the `cni` modules only the cni modules are used and not both of them. Basically, all the modules under `cni` should think of `cni` as the parent logger ?